docs/vc-http-api.yml

```diff
@@ -143,7 +143,67 @@ components:
           "proofPurpose": "assertionMethod",
           "jws": "eyJhbGciOiJFZERTQSIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..YtqjEYnFENT7fNW-COD0HAACxeuQxPKAmp4nIl8jYAu__6IH2FpSxv81w-l5PvE1og50tS9tH8WyXMlXyo45CA",
         }
-    LinkedDataProofOptions:
+    IssueCredentialOptions:
+      type: object
+      additionalProperties: false
+      description: Options for specifying how the LinkedDataProof is created.
+      properties:
+        verificationMethod:
+          type: string
+          description: The URI of the verificationMethod used for the proof. If omitted a default assertionMethod will be used.
+        proofPurpose:
+          type: string
+          description: The purpose of the proof. If omitted "assertionMethod" will be used.
+        created:
+          type: string
+          description: The date of the proof. If omitted system time will be used.
+        challenge:
+          type: string
+          description: The challenge of the proof.
+        domain:
+          type: string
+          description: The domain of the proof.
+        credentialStatusMethod:
+          type: string
+          description: The method of credential status to issue the credential including. If omitted credential status will be included.
+      example:
+        {
+          "verificationMethod": "did:example:123#z6MksHh7qHWvybLg5QTPPdG2DgEjjduBDArV9EF9mRiRzMBN",
+          "proofPurpose": "assertionMethod",
+          "created": "2020-04-02T18:48:36Z",
+          "domain": "example.com",
+          "challenge": "d436f0c8-fbd9-4e48-bbb2-55fc5d0920a8",
+          "credentialStatusMethod": "RevocationList2020Status",
```

Comment:
Yeah me neither, got an alternative in mind, `credentialStatusType` or `credentialStatus`

Reply:
personally I like `credentialStatusMethod` or `credentialStatus`

Reply:
I've always found it strange how the verification methods in the Security Vocabulary are named. Why are they called `authentication` and `assertionMethod` and `keyAgreement`. Shouldn't it be consistent, i.e. either:

- `authentication` and `assertion` and `keyAgreement`, or
- `authenticationMethod` and `assertionMethod` and `keyAgreementMethod`

Anyway, in this particular case, I think my preference would be to just re-use `credentialStatus` instead of inventing something new.

Reply:
Also... I was talking more about `RevocationList2020Status` than `credentialStatusMethod`. I'm afraid the name of the "Revocation List 2020" is wrong. It should probably be "Binary Status List 2020", and it should define properties for publishing lists that are for "suspension", "revocation", "activation", etc...

We should probably make this a map, so:

```
credentialStatus: {
  type: "RevocationList2020Status"
}
```

That would allow us to provide other options to the specific type in the future if we wanted to do so.

Reply:
I would be in favor of pushing for consistency, since I haven't shipped this to customers yet :)

Reply:
Updated to credentialStatus as an object rather than string type